docs/running.rst

```diff
@@ -205,26 +226,134 @@ interactively is started.  Use the action "help" to find out about available
 actions.
 
 
-Running :program:`supervisorctl`
---------------------------------
+:command:`supervisorctl` Actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To start :program:`supervisorctl`, run ``$BINDIR/supervisorctl``.  A
-shell will be presented that will allow you to control the processes
-that are currently managed by :program:`supervisord`.  Type "help" at
-the prompt to get information about the supported commands.
+help
 
-The :command:`supervisorctl` executable may be invoked with "one time"
-commands when invoked with arguments from a command line.  An example:
-``supervisorctl stop all``.  If arguments are present on the
-command-line, it will prevent the interactive shell from being
-invoked.  Instead, the command will be executed and ``supervisorctl``
-will exit with a code of 0 for success or running and non-zero for
-error. An example: ``supervisorctl status all`` would return non-zero
-if any single process was not running.
+   Print a list of available actions
+
+help <action>
+
+   Print help for <action>
+
+add <name> [...]
+
+   Activates any updates in config for process/group
+
+remove <name> [...]
+
+   Removes process/group from active confi
+
+update
+
+   Reload config and add/remove as necessary
+
+clear <name>
+
+   Clear a process' log files.
+
+clear <name> <name>
+
+   Clear multiple process' log files
+
+clear all
+
+   Clear all process' log files
+
+fg <process>
+
+   Connect to a process in foreground mode
+   Press Ctrl+C to exit foreground
+
+pid
+
+   Get the PID of supervisord.
+
+pid <name>
+
+   Get the PID of a single child process by name.
+
+pid all
+
+   Get the PID of every child process, one per line.
+
+restart <name>
+
+   Restart a process
+   Note: restart does not reread config files. For that, see reread and update.
```

Comment:
"reread" and "update" are both missing from this list. Are they the same?  what's the difference?  Do they imply a restart, or do you have to reread and then restart?